types/stopword/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for stopword 0.3
+// Project: https://github.com/fergiemcdowall/stopword
+// Definitions by: Rico Sandyca Novenza <https://github.com/ricosandyca>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace Stopword {
```

Comment:
Instead of using namespace.. directly export things from here. There should be constant defined in this namespace.. Instead the all methods can be exported functions and `af` like other properties as exported constants